src/utils/index.ts

```diff
@@ -0,0 +1,29 @@
+import { ContainerSize } from '@/types';
+
+export function generateRandomId() {
+  return `_${Math.random().toString(36).slice(2, 16)}`;
+}
+
+export function getSizeStyles(size?: ContainerSize) {
+  if (!size) return '';
+
+  if (size === 'full-width') {
+    return `
+      width: 100%;
+      height: auto;
+    `;
+  }
+
+  return `
+    width: ${size.width};
+    height: ${size.height};
+  `;
+}
+
+export function isThemesLoaded(themes: Object) {
+  return Object.keys(themes).length !== 0;
+}
+
+export function isEmpty(products: any[]) {
+  return products.length === 0;
+}
```

Comment:
꼭 theme, products 에만 사용되진 않을 것 같은 메서드라 좀 더 포괄적인 이름으로 바꿔줘도 좋을 것 같아요.
length의 유무는 load되어도 비어있을 수 있으니 좀 더 명확한 메서드명으로 바꿔볼까요?

```suggestion
export function hasKey(list: Object) {
  return Object.keys(list).length !== 0;
}

export function isEmptyList(list: any[]) {
  return list.length === 0;
}
```

Reply:
아하, 생각해보니 더 넓은 용도로 사용할 수 있겠군요
반영해보겠습니다!